### DISCUSSION TITLE
Extension to better export files ?

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#16454] [Feature Requests]
Is there a better module export for files ?
I would like to export a collection, including the files attached in it.
I have in mind Akeneo, which add a column with the name/path of the files, and a folder with the files.
Then it's a .zip with csv/files.

It's a bit sad we can't export files.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
